src/library_webgpu.js

```diff
@@ -660,39 +660,52 @@ var LibraryWebGPU = {
     device["pushErrorScope"](WebGPU.ErrorFilter[filter]);
   },
 
+  wgpuDevicePopErrorScope__deps: [
+    '$callUserCallback',
 #if MINIMAL_RUNTIME
-  wgpuDevicePopErrorScope__deps: ['$allocateUTF8'],
+    '$allocateUTF8OnStack',
+#else
+    '$runtimeKeepalivePush', '$runtimeKeepalivePop',
 #endif
+  ],
   wgpuDevicePopErrorScope: function(deviceId, callback, userdata) {
     var device = WebGPU.mgrDevice.get(deviceId);
+    {{{ runtimeKeepalivePush() }}}
     device["popErrorScope"]()["then"](function(gpuError) {
```

Comment:
Is there reason this "then" is in quotes like that?

Reply:
I think we just use `then()` normally everywhere else in the codebase.  I would tend of avoid being speculatively against potential closure issues and instead let testing tell is when its actually needed (but maybe I'm wrong about that approach?).